src/components/about/About.js

```diff
@@ -49,23 +48,23 @@ const About = () => {
             <img className="icons" src={categorizeIcon} alt="" />
             <h2>Categorize</h2>
             <p>
-              We have generated a model with in our data base to catergorize
-              each incident report we recive. Because our system is in beta,
-              there may be some false positives (e.g.,incident reports where law
-              enforcement were present but no use-of-force incidents actually
-              occurred), and some false negatives (e.g., incident reports where
-              use-of-force occurred and were reported but do not make it into
-              our system). To combat this, each incident report needs to be
-              approved by the Blue Witness Administrative Team.
+              We generated a model within our database to categorize each
+              incident report we receive. Our system is in beta, there may be
+              some false positives (e.g., incident reports where law enforcement
+              was present but no use-of-force incidents actually occurred).
+              Also, some false negatives (e.g., incident reports where
+              use-of-force occurred and reported but do not make it into our
+              system). To combat this, each incident report needs approval by
+              the Blue Witness Administrative Team.
             </p>
           </div>
 
           <div className="info-container">
             <img className="icons" src={confirmIcon} alt="confirm icon" />
             <h2>Confirm</h2>
             <p>
-              If a gathered incident report meets the criteria of a valid police
-              use of force incident, then it's marked to be reviewed. After an
+              If a gathered incident report meets the criteria of valid police
+              use of force, the incident will be marked and reviewed. After an
               identified incident report has been approved, it is added to the
```

Comment:
Technically the incident report exists in our database already and its status is updated to 'approved' or 'rejected'. By keeping both in the database we are able to be sure that our twitter scraper doesn't pull in a duplicated instance in our database.

Reply:
I also used this in the 'confirm' section.

Reply:
Good thinking!